<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Length On Deck:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="length">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>12.199999999999999</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Maximum Beam:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Draft:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="beam_ft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> ft</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="length_ft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> ft</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="beam">
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>3.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="draft">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="draft_ft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> ft</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="freeboard">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Freeboard</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="freeboard_ft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> ft</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QDoubleSpinBox" name="h_box">
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.900000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Beam Boxiness:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Teardrop Factor:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QDoubleSpinBox" name="teardrop">
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Draft Boxiness:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QDoubleSpinBox" name="v_box">
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.900000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="stations">
     <property name="value">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Stations:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Control Points:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="control_points">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
